### PR TITLE
docs: rearrange documentation on making PRs

### PR DESCRIPTION
**Pull request by joachim-n**
## The Issue

I can never find the PR docs because they're hiding in a page that says it's about setting up a local build and running tests, and in that page the information about PRs is split over different parts.

## How This PR Solves The Issue

- makes the link say the page is also about making PRs.
- consolidates and reorganises the bits about making PRs

### CODE REVIEW

**Review by rfay — CHANGES_REQUESTED** [review]
Looks like a nice improvement to me, thanks. Minor suggestions.

File: docs/content/developers/building-contributing.md [hunk 1]
```diff
@@ -350,18 +346,22 @@ The Docker images that DDEV uses are included in the `containers/` directory:
 
 When changes are made to an image, they have to be temporarily pushed to a tag—ideally with the same as the branch name of the PR—and the tag updated in `pkg/versionconstants/versionconstants.go`. Please ask if you need a container pushed to support a pull request.
 
-## Pull Request Pro Tips
+## Pull Requests
+
+Create a pull request on GitHub to contribute your changes back into DDEV. Once you’ve opened a pull request, a discussion will start around your proposed changes. Other contributors and users may chime in, but ultimately the decision is made by the maintainer(s). You may be asked to make some changes to your pull request. If so, add more commits to your branch and push them. They’ll automatically go into the existing pull request.
+
+If your pull request is merged, great! If not, no sweat; it may not be what the project maintainer had in mind, or they were already working on it. This happens, so our recommendation is to take any feedback you’ve received and go forth and pull request again. Or create your own open source project.
```
[the remark this answers is not in the document]
Comment (joachim-n) [reply docs/content/developers/building-contributing.md:353]: I've not changed any of that paragraph, just moved it around. Removing it feels out of scope for what I was trying to fix here.

[the remark this answers is not in the document]
Comment (rfay) [reply docs/content/developers/building-contributing.md:353]: When you muck with stuff that should be mucked with, things get spread around :) You're welcome to ignore.

[the remark this answers is not in the document]
Comment (joachim-n) [reply docs/content/developers/building-contributing.md:353]: It's obvious to people who are familiar with open source, which to be fair will probably be most people who are using DDEV in the first place. But I think that having it conveys a welcoming culture of contribution, so it does add something.

**Comment by joachim-n** [comment]
@rfay Committed you rewording suggestion and made a further change. This is ready again.

**Review by rfay — APPROVED** [review]
Works for me, thanks.

**Pull request merged**